src/api/app/services/action_build_results_service/chart_data_extractor.rb

```diff
@@ -7,6 +7,8 @@ def initialize(actions:)
     end
 
     def call
+      return [] if @actions.blank?
```

Comment:
How can this even be?

Reply:
The [request mentioned]( https://build.opensuse.org/request/show/1131616) in the exception has a action:

```
BsRequest.find_by(number: 1131616).bs_request_actions.count
=> 1
```

Reply:
For reference: https://github.com/openSUSE/open-build-service/pull/15394 and it is going to be addressed better in https://trello.com/c/JNPttW6o